src/components/property-page-widgets/Reviews/utils/getModalFormMarkup.spec.js

```diff
@@ -41,9 +41,18 @@ describe('getModalFormMarkup', () => {
 
     expect(wrapper).toMatchSnapshot();
   });
+
   it('should render the right structure when props are passed', () => {
     const wrapper = getModalForm(props);
 
     expect(wrapper).toMatchSnapshot();
   });
+
+  describe('if `isShowingPlacedholer` is `true`', () => {
```

Comment:
`isShowingPlacedholer`

```suggestion
  describe('if `isShowingPlacedholder` is `true`', () => {
```